openshift/buildenv/flower/Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-flower = "==0.9.2"
+flower = "==0.9.5"
```

Comment:
You can revert the flower changes.  We're going to just remove flower from our production rollout in app interface.  Thanks!